internal/utils/time.go

```diff
@@ -0,0 +1,10 @@
+package utils
+
+import (
+	"time"
+)
+
+func IsTimeEqualWithHysteresis(t1 time.Time, t2 time.Time, hyst time.Duration) bool {
+	diff := t1.Sub(t2).Abs()
+	return diff <= hyst
+}
```

Comment:
Usage of utils package is not a good practice. Better put this functionality in package that requires it

Reply:
This function is a too general purpose and abstract to be using only in the equipment package. Putting it to the equipment package can cause code doubling if an another package would compare time with hysteresis

Reply:
If it will be used somewhere else, then it's better to move it to a separate package with a conscious name related to functionality and avoid just putting it in utils or common. For now, I don't see any places where it would be used except this particular place.

There are a good discussions about this topic:
http://golang.org/doc/effective_go.html#package-names
https://www.reddit.com/r/golang/comments/adsqtm/avoid_package_names_like_base_util_or_common/?rdt=39410
http://blog.golang.org/package-names
https://thefortunedays.com/articles/golang-name-conventions/

Reply:
Agreed. Refactored the code